Ical.Net/Calendar.cs

```diff
@@ -143,20 +147,54 @@ public override int GetHashCode()
     public virtual ICalendarObjectList<VTimeZone> TimeZones => _mTimeZones;
 
     /// <summary>
-    /// A collection of <see cref="Todo"/> components in the iCalendar.
+    /// A collection of <see cref="CalendarComponents.Todo"/> components in the iCalendar.
     /// </summary>
     public virtual IUniqueComponentList<Todo> Todos => _mTodos;
 
+    /// <summary>
+    /// Gets or sets the version of the iCalendar definition. The default is <see cref="LibraryMetadata.Version"/>
+    /// as per RFC 5545 Section 3.7.4 and must be specified.
+    /// <para/>
+    /// It specifies the identifier corresponding to the highest version number of the iCalendar specification
+    /// that is required in order to interpret the iCalendar object.
+    /// <para/>
+    /// <b>Do not change unless you are sure about the consequences.</b>
+    /// <para/>
+    /// The default value does not apply to deserialized objects.
+    /// </summary>
     public virtual string Version
     {
         get => Properties.Get<string>("VERSION");
-        set => Properties.Set("VERSION", value);
+        set
+        {
+            if (string.IsNullOrEmpty(value))
```

Comment:
Just a nit - not sure we need a special handling for null and empty here. Wouldn't know why for instance "XXX" would be any better than "". So for simplicity and to reduce the work f or CodeCov (aka the number of branches) I'd rather avoid such checks where not explicitly necessary.